ext/pcntl/pcntl.c

```diff
@@ -1262,7 +1262,7 @@ PHP_FUNCTION(pcntl_unshare)
 		switch (errno) {
 #ifdef EINVAL
 			case EINVAL:
-				zend_argument_value_error(1, "must be a combination of CLONE_* flags");
+				zend_argument_value_error(1, "must be a combination of CLONE_* flags or at least one flag is unsupported by the kernel");
```

Comment:
This is a super tiny nit, but I think it would be better to place a comma before `or` to make it clear it's a different clause in the sentence.